File: server/aoNewsletter2/Models/Db/DesignBlockThemeModel.vb

Imports Contensive.Models.Db

Namespace Models
    Public Class DesignBlockThemeModel
        Inherits DbBaseModel

        Public Shared ReadOnly Property tableMetadata As DbBaseTableMetadataModel = New DbBaseTableMetadataModel("Design Block Themes", "dbThemes", "default", False)

    End Class
End Namespace

File: server/aoNewsletter2/Models/Db/DesignBlockThemeModel.cs

using Contensive.Models.Db;

namespace Contensive.Addons.Newsletter.Models {
    public class DesignBlockThemeModel : DbBaseModel {

        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Design Block Themes", "dbThemes", "default", false);

    }
}